mitodl/ocw-studio
# migrate ocw-www content type pages to page

#### Pre-Flight checklist


#### What are the relevant tickets?
#1311 

#### What's this PR do?
This PR migrates the content type value for pages in ocw-www from `pages` to `page`. This goes along with changes from https://github.com/mitodl/ocw-hugo-projects/pull/161.

#### How should this be manually tested?
1. On master, check the Pages listing of `ocw-www`. There should be none.
2. Check out this branch and `docker-compose restart web`.
3. Check the Pages listing of `ocw-www` again. Now there should be pages.
